GeneralsMD/Code/GameEngine/Source/Common/System/SaveGame/GameState.cpp

```diff
@@ -1623,7 +1624,7 @@ void GameState::xfer( Xfer *xfer )
 	{
 		char string[ _MAX_PATH ];
 
-		strcpy( string, TheGlobalData->m_mapName.str() );
+		strlcpy(string, TheGlobalData->m_mapName.str(), ARRAY_SIZE(string));
```

Comment:
This string copy is unnecessary. We can use `TheGlobalData->m_mapName` directly with `AsciiString::reverseFind`.